pages/Page.vue

```diff
@@ -1,7 +1,158 @@
-<script lang="ts" setup></script>
+<script lang="ts">
+//var page_name="something"
+//ask taz about type/data declaration
+//export default 
+//onMounted(() => {
+
+//})
+type Page ={
+    Cuid: String,
+    url : String,
+    page_name: String,
+    day_of_birth: Date,
+    day_of_passing: Date,
+    visitation_date: Date,
+    visitation_location: String,
+    timezone: String,
+    visitation_description: String,
+    funeral_date: Date,
+    funeral_description: String,
+    funeral_location: String,
+    obituary: String,
+    deadline: Date,
+    donation_goal: Number,
+    donated_amount: Number,
+    donated_percentage: Number,
+    query: String
+
+}
+const data = ref<Page>( {
+    Cuid : "",
+    url : "",
+    page_name: "",
+    day_of_birth: new Date(),
+    day_of_passing: new Date(),
+    visitation_date: new Date(),
+    visitation_location: "",
+    timezone: "",
+    visitation_description: "",
+    funeral_date: new Date(),
+    funeral_description: "",
+    funeral_location: "",
+    obituary: "",
+    deadline: new Date(),
+    donation_goal: 0,
+    donated_amount:0,
+    donated_percentage: 0,
+    query: ""
+}) 
+const create_checkup_session =async () => {
+    await useFetch('/api/page_donation',{
+    method:'POST'
+})
+const page_data_cookie=useCookie('cv');
+const page_data= computed (() => JSON.parse(page_data_cookie.value || "{}"))
+const router=useRoute()
+const id= computed(() => {router.params.id});
+
+	/* update the progress bar based on donation amount !!!!!
+	const progress=document.querySelector('.progress') || "{}";
+	progress.style.width=progress.getAttribute('donated-amount') + "%";
+	progress.style.opacity=1;
+    */
+	
+}
+
+</script>
 
 <template lang="pug">
-div This is where a page will be rendered. Use route or query parameters to identify which page needs to be loaded.
+doctype html 
+head 
+	link(rel='preconnect' href='https://fonts.googleapis.com/')
```

Comment:
Don't need doctype, html, head stuff here - this is provided by Nuxt in a separate location

Reply:
done